docs/data-ai/capture-data/advanced/advanced-data-capture-sync.md

```diff
@@ -0,0 +1,626 @@
+tags: ["data management", "data", "services"]
+weight: 10
+layout: "docs"
+type: "docs"
+platformarea: ["data"]
+description: "Advanced data capture and data sync configurations."
+prev: /data-ai/capture-data/conditional-sync/
+date: "2025-02-10"
+---
+
+You can configure both the data management and data capture with attributes in JSON for advanced use cases.
+You can also configure data capture for remote parts.
+
+### Advanced data management service configurations
+
+To configure the data manager in JSON, see the following example configurations:
+
+{{< tabs >}}
+{{% tab name="viam-server" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [],
+  "services": [
+    {
+      "name": "my-data-manager",
+      "namespace": "rdk",
+      "type": "data_manager",
+      "attributes": {
+        "sync_interval_mins": 1,
+        "capture_dir": "",
+        "tags": [],
+        "capture_disabled": false,
+        "sync_disabled": true,
+        "delete_data_on_part_deletion": true,
+        "delete_every_nth_when_disk_full": 5,
+        "maximum_num_sync_threads": 250
+      }
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% tab name="viam-micro-server" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [],
+  "services": [
+    {
+      "name": "my-data-manager",
+      "namespace": "rdk",
+      "type": "data_manager",
+      "attributes": {
+        "capture_dir": "",
+        "tags": [],
+        "additional_sync_paths": [],
+        "sync_interval_mins": 3
+      }
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The following attributes are available for the data management service:
+
+{{< expand "Click to view data management attributes" >}}
+
+<!-- prettier-ignore -->
+| Name               | Type   | Required? | Description | `viam-micro-server` Support |
+| ------------------ | ------ | --------- | ----------- | ------------------- |
+| `capture_disabled` | bool   | Optional | Toggle data capture on or off for the entire machine {{< glossary_tooltip term_id="part" text="part" >}}. Note that even if capture is on for the whole part, but is not on for any individual {{< glossary_tooltip term_id="component" text="components" >}} (see Step 2), data is not being captured. <br> Default: `false` | <p class="center-text"><i class="fas fa-check" title="yes"></i></p> |
+| `capture_dir`      | string | Optional | Path to the directory on your machine where you want to store captured data. If you change the directory for data capture, only new data is stored in the new directory. Existing data remains in the directory where it was stored. <br> Default: `~/.viam/capture` | <p class="center-text"><i class="fas fa-check" title="yes"></i></p> |
+| `tags` | array of strings | Optional | Tags to apply to all images or tabular data captured by this machine part. May include alphanumeric characters, underscores, and dashes. | <p class="center-text"><i class="fas fa-times" title="no"></i></p> |
```

Comment:
I think we want no more "no" marks so remove?

Also, the checks and Xs are showing up very small within the expander.

Reply:
```suggestion
| `tags` | array of strings | Optional | Tags to apply to all images or tabular data captured by this machine part. May include alphanumeric characters, underscores, and dashes. |  |
```